account_bank_receipt/data/sequence.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
```

Comment:
I think you don't need to copy all the credit from all xml files.
However, you can add comment in Description section of readme, that this module is based on the work of account_check_deposit, but make it less specific to cover more use case.